### PR TITLE
File Added : Assigmt6-2, Assigment 7&8

### PR DESCRIPTION
**Pull request by Amit727619**

### CODE REVIEW

File: amit/html/Assigment6/Amit6_2.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+                <label for="age">Oracle</label><br /><br /><br />
+                <label for="highest qualification"><b>Highest Qualification:</b></label>
+                <select name="qualification" id="qualification">
+                    <option value="diploma">Diploma in Computer Science</option>
+                    <option value="science">Science Graduate</option>
+                    <option value="Bachelor in Engg">Bachelor in Engg</option>
+                    <option value="Masters">Masters in Computer Science</option>
+                    <option value="Masters">Masters in Engg</option>
+                    <option value="other">Other</option></select
+                ><br /><br /><br />
```
Comment (Pawan1005) [inline amit/html/Assigment6/Amit6_2.html]: > this need to be on same line

File: amit/html/Assigment6/Amit6_2.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                <label for="University"><b>University</b></label>
+                <input list="browsers" name="browser" />
+                <datalist id="browsers">
+                    <option value="Software Training"></option>
+                    <option value="Networking"></option>
+                    <option value="Management"></option>
+                    <option value="Marketing"></option></datalist
+                ><br /><br />
```
Comment (Pawan1005) [inline amit/html/Assigment6/Amit6_2.html]: same here also > this need to be with same line

File: amit/html/Assigment7/login.html [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+            width: auto;
+            }
+        h1 {
+            text-align: left;
+        }
+        .container{
+              text-align: center;
+        }
+    </style>
+</head>
```
Comment (Pawan1005) [inline amit/html/Assigment7/login.html]: indentation missing

File: amit/html/Assigment8/Amit8..html [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+                float: left;
+            }
+            .e {
+                position: absolute;
+                top: 800px;
+                right: 10px;
+                float: right;
+                width: 70%;
+                padding: auto;
+            }
```
Comment (Pawan1005) [inline amit/html/Assigment8/Amit8..html]: give proper class names

File: amit/html/Assigment8/Amit8..html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+            us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+            us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact
+            us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact
+            us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+            us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+            us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+            us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline amit/html/Assigment8/Amit8..html]: add new line at the end of file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

File: amit/html/Assigment7/login.html [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+            <a href="home.html"><button>home </button></a> 
+            <a href="personaldetails.html"><button >Personal Detailes</button></a>
+            <a href="Professional_Details.html"><button >Professional Details</button></a>   
+            <a href="Aboutus.html"><button >Aboutus</button></a>   
+            <a href="Contactus.html"><button >Contactus</button></a>   
+            <a href="login.html"><button >Login</button></a>
+        </div>
+        </div>
+        <div class = "container">
+            <label for="uname"><b>Username</b></label>
```
Comment (shivling123) [inline amit/html/Assigment7/login.html]: for attribute given to lable and id given to input filed must be same

make this changes in all assesment

File: amit/html/Assigment7/personaldetails.html [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+                <a href="Aboutus.html"><button>Aboutus</button></a>
+                <a href="Contactus.html"><button>Contactus</button></a>
+                <a href="login.html"><button>Login</button></a>
+            </div>
+        </div>
+        <div class="matter">
+            <h1>Personal Details</h1>
+            <form>
+                <label for="name">First Name</label>
+                <input type="text" id="first name" name="first name" /><br /><br />
```
Comment (shivling123) [inline amit/html/Assigment7/personaldetails.html]: for attribute given to lable and id given to input filed must be same

make this changes in all assesment

**Review by shivling123 — CHANGES_REQUESTED** [review]
do required changes as per comment 

upload again asap

**Review by pritikekan — APPROVED** [review]

File: amit/html/Assigment7/login.html [hunk 8]
```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+<html>
+    <meta charset="UTF-8">
```
Comment (Pawan1005) [inline amit/html/Assigment7/login.html]: head tag opening is missing

File: amit/html/Assigment8/Amit8_1html [hunk 9]
```diff
@@ -0,0 +1,94 @@
...
+            us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+            us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact
+            us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact
+            us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+            us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+            us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact
+            us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us.Contact us. Contact us.Contact us.Contact us.Contact us.Contact us.
+        </div>
+    </body>
+</html>
```
Comment (Pawan1005) [inline amit/html/Assigment8/Amit8_1html]: check extension of file

**Review by Pawan1005 — CHANGES_REQUESTED** [review]

**Review by Pawan1005 — APPROVED** [review]

**Pull request merged**